src/features/machines/depositGenerateAddressMachine.ts

```diff
@@ -7,7 +7,8 @@ export type DepositGeneratedDescription = {
 }
 
 type Context = {
-  accountId: string | null
+  userAddress: string
+  userChainType: ChainType
```

Comment:
**JMT:** Prefix `user` makes `ChainType` more specific but still I feel some misleading.

Reply:
But when it comes together with `userAddress` it gives me more sense, so all is good.